### PR TITLE
Test Case for MergeMaps

### PR DESCRIPTION
**Pull request by johnhooks3**
Add test case for mergeMaps that tests different types being passed
in through the empty interface.

### CODE REVIEW

File: cmd/common_test.go [hunk 1]
```diff
@@ -0,0 +1,33 @@
+package cmd
```
Comment (madhokie) [inline cmd/common_test.go:1]: Add file header copyright comment block

**Review by madhokie — CHANGES_REQUESTED** [review]
`Makefile` needs to have `test` target added to `local:` and `release`... i.e.

`local: test`
`release: test windows linux darwin`

File: cmd/common_test.go [hunk 2]
```diff
@@ -0,0 +1,47 @@
+// Copyright © 2019 BoxBoat engineering@boxboat.com
```
Comment (boxboatmatt) [inline cmd/common_test.go:1]: 2020

Comment (johnhooks3) [reply cmd/common_test.go]: Sorry, fixed.

**Pull request merged**